apkid/rules/dex/obfuscators.yara

```diff
@@ -109,3 +109,28 @@ rule bitwise_antiskid : obfuscator
     is_dex and
     any of them
 }
+
+rule arxan : obfuscator
+{
+  meta:
+    description = "Arxan Dalvik"
```

Comment:
Could you add a link to a website describing what this is?
Also, please include a reference sample hash.

Reply:
It seems that @strazzere can provide more insight here. I will find another sample to share with you guys.
Description: https://www.arxan.com/products/application-protection-mobile/

Reply:
Thanks, that's perfect. Could you add this as a comment above description?

Reply:
done!